### DISCUSSION TITLE
how to disablde Applied Sorting

### DISCUSSION THREAD

**Discussion by dafuku** [rappasoft/laravel-livewire-tables#570] [Q&A] [answered]
Hot to disable Applied Sorting .
I want to use only single column sorting .

Like jquery datatables sorting.
[https://datatables.net/](https://datatables.net/)

Thank you.

![Applied Sorting](https://user-images.githubusercontent.com/19335127/143021844-95c256f8-3034-47b6-891d-7e3245ecb1f7.png)

**Comment by rappasoft** [comment, marked as answer]
Use this property: https://github.com/rappasoft/laravel-livewire-tables/blob/master/src/Traits/WithSorting.php#L15

**Reply by dafuku** [reply]
Mr rappasoft 
Thank you !!
Very Nice Answer !
I have solved the problem. 😊 👍
